frontend/src/app/services/product/product.service.ts

```diff
@@ -1,34 +1,51 @@
+import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
-import { BehaviorSubject, map, Observable, of} from 'rxjs';
+import { BehaviorSubject, map, Observable } from 'rxjs';
 import { Product } from 'src/app/interfaces/product';
-import { PRODUCTS } from 'src/app/mocks/mock-products';
+import { environment } from 'src/environments/environment';
 import { SortService } from '../sort/sort.service';
 
 @Injectable({
   providedIn: 'root'
 })
-export class ProductService{
-  public products: Product[] = PRODUCTS;
-  public products$: BehaviorSubject<Product[]> = new BehaviorSubject<Product[]>(this.products);
+
+export class ProductService {
+  private productApi: string = environment.apiUrl;
```

Comment:
Why baseurl from env names productApi. It is wrong. productApi it is "${this.productApi}/products"